src/System.Runtime/tests/System/DecimalTests.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Tests
 {
+    [Collection("CultureTestCollection")]
```

Comment:
Wait, though, CurrentCulture is specific to a thread... as long as the operation is synchronous and reset the culture, why is this a problem?